app/src/main/java/com/simpragma/recipe/recipeapp/DatabaseHandler.java

```diff
@@ -0,0 +1,127 @@
+package com.simpragma.recipe.recipeapp;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import com.simpragma.recipe.recipeapp.ResultList;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Madvesha on 2/4/18.
+ */
+
+public class DatabaseHandler extends SQLiteOpenHelper {
+
+    // All Static variables
+    // Database Version
+    private static final int DATABASE_VERSION = 2;
+    // Database Name
+    private static final String DATABASE_NAME = "recipeManager";
+    // Recipe table name
+    private static final String TABLE_RECIPE = "recipeitems";
+
+    String TAG = getClass().getSimpleName();
+
+    // Recipe Table Columns names
+    private static final String KEY_ID = "id";
+    private static final String KEY_TITLE = "title";
+    private static final String KEY_INGREDIENTS = "ingredients";
+    private static final String KEY_THUMBNAIL = "thumbnail";
+    private static final String KEY_HREF = "href";
+
+    public DatabaseHandler(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        //Creating  Recipe Table
+        Log.d(TAG, "Creating Recipe Table");
+        String CREATE_RECIPE_TABLE = "CREATE TABLE " + TABLE_RECIPE + "("
+                + KEY_ID + " INTEGER PRIMARY KEY," + KEY_TITLE + " TEXT,"
+                + KEY_INGREDIENTS + " TEXT,"
+                + KEY_HREF + " TEXT,"
+                + KEY_THUMBNAIL + " TEXT" + ")";
+        db.execSQL(CREATE_RECIPE_TABLE);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+
+        Log.d(TAG, "in onUpgrade. Old is: " + oldVersion + " New is: " + newVersion);
+
+        if (newVersion > oldVersion) {
+            // db.execSQL("ALTER TABLE Recipe ADD COLUMN NOTES TEXT");
+            Log.d(TAG, "If any change alteration in table above table excute");
```

Comment:
What is happening here? We are ignoring the upgrade altogether?

Reply:
Here i am not add any column to table. For that just put log